### PR TITLE
[release] Adjust website for Kubernetes operator 1.8.0 release

### PR DESCRIPTION
**Pull request by mxm**
Draft PR for the upcoming Kubernetes operator 1.8.0 release

### CODE REVIEW

**Review by 1996fanrui — COMMENTED** [review]
We need to prepare the release announcement in this PR as well, right?

**Comment by mxm** [comment]
Yes, the blog post is still a TODO, although it isn't strictly needed for the release.

**Comment by mxm** [comment]
@1996fanrui @gyfora I added the release post.

File: docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+A common request is to support a streamlined, user-friendly way of redeploying from a target savepoint. Previously this
+was only possible by deleting the CR and recreating it with initialSavepointPath. A big downside of this approach is a
+loss of savepoint/checkpoint history in the status that some platforms may need, resulting in non-cleaned up savepoints.
+
+We introduced a `savepointRedeployNonce` field in the job spec similar to other action trigger nonces.
+
+If the nonce changes to a new non-null value the job will be redeployed from the path specified in the
+initialSavepointPath (or empty state If the path is empty).
+
+### Cluster shutdown and resource cleanup improvements:
```
Comment (gyfora) [inline docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md]: Remove `:`

File: docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+The release includes many improvements to the operator core, the autoscaler, and introduces new features
+like TaskManager memory auto-tuning.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### Flink Autotuning
```
Comment (gyfora) [inline docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md:29]: Should we call this `Flink Memory Autotuning` ?

Comment (gyfora) [reply docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md:29]: I see you have reference of other config tuning in the future, it is also okay to keep it like this :)

**Review by gyfora — APPROVED** [review]
Looks great @mxm , I added a very small correction

File: docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md (hunk 2 above)
Comment (mxm) [reply docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md:29]: I had thought about that as well but decided to market it as a more general feature which hosts various functionality.

**Comment by mxm** [comment]
@1996fanrui I'm going to merge. Please feel free to comment on the PR. We can still correct any mistakes or further improve the post!

**Comment by 1996fanrui** [comment]
> @1996fanrui I'm going to merge. Please feel free to comment on the PR. We can still correct any mistakes or further improve the post!

Sorry for the late response! `1.7.0` is the first version for `Autoscaler Standalone`, and we finished a series of improvements in `1.8.0`, such as : 

- Autoscaler Standalone control loop supports multiple thread
- Implement JdbcAutoScalerStateStore
- Implement JdbcAutoScalerEventHandler
- etc

After these improvements, the `Autoscaler Standalone` is close to production ready. Would you mind if we mention them in this release announcement?

**Comment by mxm** [comment]
Of course! For reference, follow-up PR is here: #728

**Pull request merged**